ui/src/components/option-group/QOptionGroup.json

```
@@ -15,6 +15,31 @@
     "options": {
       "type": "Array",
       "desc": "Array of objects with value and label props. The binary components will be created according to this array; Props from QToggle, QCheckbox or QRadio can also be added as key/value pairs to control the components singularly",
+      "definition": {
+        "label": {
+          "type": "String",
+          "desc": "Label",
+          "required": true,
+          "examples": [ "Option 1", "Option 2", "Option 3" ]
+        },
+        "value": {
+          "type": "Any",
+          "desc": "Value",
+          "required": true,
+          "examples": [ "op1", "op2", "op3" ]
+        },
+        "disable": {
+          "type": "Boolean",
+          "desc": "Disable",
+          "default": false,
+          "examples": [ true, false ]
+        },
+        "...props": {
+          "type": "Any",
+          "desc": "Any other QOptionGroup props",
+          "__exemption": [ "examples" ]
```

`examples` do not have any examples of props from other components(_QToggle, QCheckbox, or QRadio_). Please try to add some.

---

What especially is this `"__exemption"`? I copied it over from somewhere else :sweat_smile:  
I just discovered that it is possible to add some custom props like in this example: https://quasar.dev/vue-components/option-group#example--label-slots  

So should it be this?

```suggestion
          "__exemption": [ "icon", "color" ]
```

---

In the associated JSON file, `__exception` means the targeted props are not required to have an example.

---

Let's say you have a function definition, which has `examples`, and for a specific param of it, you think the `examples` from the function are already enough to describe it, so you can add `examples` to `__exemption` to skip the need of adding an example for that specific param.

However, in this case, `examples` of `options` are not descriptive enough, so you shouldn't skip defining an example. So, you would need to remove the `__exemption` completely and add `examples` prop with some values like `"examples": [ "whatever=\"foo-bar\"" ]` but with a better example than `whatever` 😄.

---

```suggestion
          "examples": [ "val=\"car\"", ":true-value=\"trueValue\"", "checked-icon=\"visibility\"" ]
```
I think these examples should be clear enough.